client/src/Components/Layout/AdminEventDetails.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import { Grid, Box, Typography, Button, Paper } from '@material-ui/core';
+import { Redirect } from 'react-router-dom';
+import {
+  QueryBuilder as QueryBuilderIcon,
+  Room as RoomIcon,
+  EventNote as EventNoteIcon,
+} from '@material-ui/icons';
+import axios from 'axios';
+import Cookies from 'js-cookie';
+import { withStyles } from '@material-ui/core/styles';
+import { red } from '@material-ui/core/colors';
+import LoaderProgress from './../Common/LoaderProgress';
+import '../../Theme/Css/App.css';
+
+const useStyles = (theme) => ({
+  root: { 'text-align': 'center' },
+  red: {
+    color: red[500],
+  },
+  btnCancel: {
+    borderRadius: 15,
+    width: 230,
+    height: 33,
+    background: '#FBF3F1',
+    border: '1px solid #F6554D',
+    fontWeight: 900,
+    fontSize: 16,

```

Comment:
Same with font sizes, it's good to have a specific set of 4-6 font sizes that you can use and stick to them